osu.Game/Database/IModelManager.cs

```diff
@@ -18,16 +17,14 @@ public interface IModelManager<TModel>
         where TModel : class
     {
         /// <summary>
-        /// A bindable which contains a weak reference to the last item that was updated.
-        /// This is not thread-safe and should be scheduled locally if consumed from a drawable component.
```

Comment:
Would probably have kept this comment here, as it is in `IModelDownloader`.

Reply:
I was going to mention that too but the implementation still has the comment, and unsure if this event is going to live long, so I let it go.

Reply:
Wasn't an intentional removal. In theory this won't exist forever, but that said... basically everywhere we use `event` has this clause applied so I'm not sure how meaningful it is to have on each usage..